modules/serverbidBidAdapter.js

```diff
@@ -125,84 +86,122 @@ ServerBidAdapter = function ServerBidAdapter() {
-    }
+    });
 
-    if (data.placements.length) {
-      ajax(config.BASE_URI, _responseCallback, JSON.stringify(data), { method: 'POST', withCredentials: true, contentType: 'application/json' });
-    }
-  }
+    ret.data       = JSON.stringify(data);
+    ret.bidRequest = validBidRequests;
+    ret.url        = ENDPOINT_URL;
+
+    return ret;
+  },
 
-  function _responseCallback(result) {
+  /**
+         * Unpack the response from the server into a list of bids.
+         *
+         * @param {*} serverResponse A successful response from the server.
+         * @return {Bid[]} An array of bids which were nested inside the server.
+         */
+  interpretResponse: function(serverResponse, bidRequest) {
     let bid;
+    let bids;
     let bidId;
     let bidObj;
-    let bidCode;
-    let placementCode;
+    let bidResponses = [];
 
-    try {
-      result = JSON.parse(result);
-    } catch (error) {
-      utils.logError(error);
-    }
+    bids = bidRequest.bidRequest;
 
-    for (let i = 0; i < bidIds.length; i++) {
-      bidId = bidIds[i];
-      bidObj = utils.getBidRequest(bidId);
-      bidCode = bidObj.bidder;
-      placementCode = bidObj.placementCode;
+    for (let i = 0; i < bids.length; i++) {
+      bid = {};
+      bidObj = bids[i];
+      bidId = bidObj.bidId;
+      bid.bidderCode = bidObj.bidder;
 
-      if (result) {
-        const decision = result.decisions && result.decisions[bidId];
+      if (serverResponse) {
+        const decision = serverResponse.decisions && serverResponse.decisions[bidId];
         const price = decision && decision.pricing && decision.pricing.clearPrice;
 
         if (decision && price) {
-          bid = bidfactory.createBid(1, bidObj);
-          bid.bidderCode = bidCode;
+          bid.requestId = bidId;
           bid.cpm = price;
           bid.width = decision.width;
           bid.height = decision.height;
           bid.ad = retrieveAd(decision);
-        } else {
-          bid = bidfactory.createBid(2, bidObj);
-          bid.bidderCode = bidCode;
+          bid.currency = 'USD';
+          bid.creativeId = decision.adId;
+          bid.ttl = 360;
+          bid.netRevenue = true;
+          bid.referrer = utils.getTopWindowUrl();
+
+          bidResponses.push(bid);
         }
-      } else {
-        bid = bidfactory.createBid(2, bidObj);
-        bid.bidderCode = bidCode;
       }
-      bidmanager.addBidResponse(placementCode, bid);
     }
-  }
 
-  function retrieveAd(decision) {
-    return decision.contents && decision.contents[0] && decision.contents[0].body + utils.createTrackPixelHtml(decision.impressionUrl);
-  }
-
-  function getSize(sizes) {
-    const result = [];
-    sizes.forEach(function(size) {
-      const index = sizeMap.indexOf(size[0] + 'x' + size[1]);
-      if (index >= 0) {
-        result.push(index);
-      }
-    });
-    return result;
+    return bidResponses;
+  },
+
+  getUserSyncs: function(syncOptions) {
+    if (syncOptions.iframeEnabled) {
```

Comment:
We can just make this return empty array for now. Or if feeling ambitious figure out a way to get it to render the smart sync code.